.devcontainer/base_with_next/devcontainer.json

```diff
@@ -31,12 +31,9 @@
     "8027": {
       "label": "Mailhog"
```

Comment:
what about this, in my tests "ddev status" and "docker ps" has not shown it, so if it's not included, it should not be here.

Reply:
I cheked it and it worked